docs/sphinx/user_guide/feature/policies.rst

```diff
@@ -125,9 +125,24 @@ The following policies may only be used with the ``RAJA::kernel`` method.
 CUDA Policies
 ^^^^^^^^^^^^^^
 
-* ``cuda_block_exec`` - Map loop iterations to CUDA thread blocks.
-* ``cuda_thread_exec`` - Map loop iterations to CUDA threads in a thread block.
-* ``cuda_threadblock_exec<BLOCK_SIZE>`` - Map loop iterations to CUDA thread blocks, each with given block size number of threads.
+* ``cuda_thread_x_direct`` - Direct mapping of loop iterations to cuda threads in the x dimension.
+* ``cuda_thread_y_direct`` - Direct mapping of loop iterations to cuda threads in the y dimension.
+* ``cuda_thread_z_direct`` - Direct mapping of loop iterations to cuda threads in the z dimension.
+  
+  * If multiple thread direct policies are used within kernel; the product of the sizes must be :math:`\leq` 1024.
+  * Repeating thread direct policies with the same thread dimension in perfectly nested loops is not supported.
+  * Thread direct policies are only recommended with certain loop patterns such as tiling.
+
+* ``cuda_thread_x_loop`` - Similar to thread direct policies, but introduces a grid-stride loop over cuda blocks in the x dimension.
+* ``cuda_thread_y_loop`` - Similar to thread direct policies, but introduces a grid-stride loop over cuda blocks in the y dimension.
+* ``cuda_thread_z_loop`` - Similar to thread direct policies, but introduces a grid-stride loop over cuda blocks in the z dimension.
```

Comment:
maybe this should be a {block size in the xyz dimension} stride loop instead of {grid} stride as grid refers to the entire grid instead of just the threads of a block